framework/dbt/basic/macros/overrides.sql

```diff
@@ -1,116 +0,0 @@
-{% macro default__reset_csv_table(model, full_refresh, old_relation, agate_table) %}
-    {% set sql = "" %}
-    {% if full_refresh %}
-        {{ adapter.drop_relation(old_relation) }}
-        {% set sql = create_csv_table(model, agate_table) %}
-    {% else %}
-        {{ adapter.truncate_relation(old_relation) }}
-        {% set sql = "delete from " ~ old_relation %}
-    {% endif %}
-
-    {{ return(sql) }}
-{% endmacro %}
-
```

Comment:
Refactored per https://github.com/crate/dbt-cratedb2/pull/2/commits/3385045ce9642b24ef650a3f5602c284c614c26c. Not in use, yet.